src/sentry/templates/sentry/includes/sdk-config.html

```diff
@@ -1,10 +0,0 @@
-{% load sentry_api %}
-{% load sentry_dsn %}
-<script>
-  window.__SENTRY__OPTIONS = {
-    dsn: '{% public_dsn %}',
-    release: '{{ sentry_version.build }}',
-    whitelistUrls: {% convert_to_json ALLOWED_HOSTS %}
-  };
-  window.__SENTRY__USER = {% get_user_context request %} || undefined;
-</script>
```

Comment:
Removed since this is inlined. This file was overriden in getsentry, but see the PR there for how it's been changed.